src/vdaf/prio3.rs

```diff
@@ -159,6 +162,34 @@ impl Prio3Aes128Histogram {
     }
 }
 
+/// The average type. Each measurement is an integer in `[0,2^bits)` for some `0 < bits < 64` and the
+/// aggregate is the arithmetic average. This type is a test for demonstrating that the number of measurements
+/// may be used in the `decode_result()` function. No validity checks are done on the input data.
+#[cfg(test)]
+pub type TestPrio3Aes128Average = Prio3<Average<Field128>, PrgAes128, 16>;
```

Comment:
If this type is indeed only for testing, then it need not be `pub`; `pub(crate)` would be sufficient. However I think it might be worth exposing in the public API. In this case, we could drop the `Test` prefix.